prow/pjutil/pjutil_test.go

```diff
@@ -750,12 +750,12 @@ func TestJobURL(t *testing.T) {
 					PathStrategy: prowapi.PathStrategyExplicit,
 				}},
 			}},
-			expected: "https://gubernator.com/build/bucket/pr-logs/pull/org_repo/1",
+			expected: "https://gubernator.com/build/gs/bucket/pr-logs/pull/org_repo/1",
 		},
 		{
-			name: "decorated job with prefix uses gcsupload and new bucket format with gcs",
+			name: "decorated job with prefix uses gcsupload and new bucket format with gcs (deprecated job url format)",
 			plank: config.Plank{
-				JobURLPrefixConfig: map[string]string{"*": "https://prow.k8s.io/view/gcs"},
```

Comment:
Can we keep a testcase without the trailing slash?

Reply:
Yup I'll add another. Although I'm pretty sure it doesn't actually work if the trailing slash is missing I had problems a while ago when I forgot it. But maybe that was just a strange s3 fork version of mine :)

Reply:
Okay and of course that was the case that was broken because of TrimSuffix only trimmed "gcs/" not "gcs"